subprojects/core/src/main/java/org/gradle/api/initialization/Settings.java

```diff
@@ -208,4 +209,20 @@
      */
     @Incubating
     void buildCache(Action<? super BuildCacheConfiguration> action);
+
+    /**
+     * Configures plugin management.
+     *
+     * @since 3.5
+     */
+    @Incubating
+    void pluginManagement(Action<? super PluginManagementSpec> pluginManagementSpec);
```

Comment:
That's a good point, but I think out of scope for this change. Could you open a separate issue like "Remove Action<> methods in favor of annotation"?